src/store/modules/order/OrderState.ts

```diff
@@ -0,0 +1,6 @@
+export default interface OrderState {
+    inProgress: {
+      list: any,
+      total: number
```

Comment:
When storing the response in status we are storing the total in the `inProgress.list.total` then why we need this total in state?

Reply:
Updated the orders state.